docs/troubleshooting/large-partition-table.rst

```diff
@@ -73,9 +75,9 @@ Example output:
 
 .. code-block:: console
 
-   keyspace_name | table_name | sstable_name     | partition_size | partition_key                                       | range_tombstones | rows | compaction_time
-   --------------+------------+------------------+----------------+-----------------------------------------------------+------------------+------+---------------------
-          demodb |       tmcr | md-6-big-Data.db |     1188716932 | {key: pk{000400000001}, token:-4069959284402364209} |                0 | 1942 | 2018-07-23 08:10:34
+   keyspace_name | table_name | sstable_name     | partition_size | partition_key                                       | dead_rows | range_tombstones | rows | compaction_time
```

Comment:
please try the query in the example (SELECT *) on real data and make sure the example output reflects the same order of columns.

Reply:
I have changed the order of the columns to reflect the order in which the `SELECT * FROM system.large_partitions;` lists the values.